packages/ui/src/proposals/components/VotesPreview/VotesPreview.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { ProposalVoteKind } from '@/common/api/queries'
+import { ButtonGhost } from '@/common/components/buttons'
+import { DropDownButton, DropDownToggle, ToggleContainer } from '@/common/components/buttons/DropDownToggle'
+import { FileIcon } from '@/common/components/icons/FileIcon'
+import { RowGapBlock } from '@/common/components/page/PageContent'
+import { TextInlineMedium } from '@/common/components/typography'
+import { BorderRad, Colors } from '@/common/constants'
+import { plural } from '@/common/helpers'
+import { useToggle } from '@/common/hooks/useToggle'
+import { isDefined } from '@/common/utils'
+import { spacing } from '@/common/utils/styles'
+import { MemberInfo } from '@/memberships/components'
+import { ProposalVotes } from '@/proposals/hooks/useProposalVotes'
+import { ProposalVote } from '@/proposals/types'
+
+interface VotesPreviewProps {
+  votes: ProposalVotes
+}
+
+const { Approve, Reject, Slash, Abstain } = ProposalVoteKind
+
+export const VotesPreview = ({ votes }: VotesPreviewProps) => (
+  <VotesContainer>
+    <VotePreview kind={Approve} count={votes.count.approve} votes={votes.map.get(Approve)} />
+    <VotePreview kind={Reject} count={votes.count.reject} votes={votes.map.get(Reject)} />
+    <VotePreview kind={Slash} count={votes.count.slash} votes={votes.map.get(Slash)} />
+    <VotePreview kind={Abstain} count={votes.count.abstain} votes={votes.map.get(Abstain)} />
+    <VotePreview kind="Not Voted" count={votes.count.remain} />
+  </VotesContainer>
+)
+
+interface VotePreviewProps {
+  kind: string
+  count?: number
+  votes?: ProposalVote[]
+}
+
+export const VotePreview = ({ kind, count, votes }: VotePreviewProps) => {
+  const [isOpen, toggle] = useToggle()
+
+  return (
+    <VoteType>
+      <VoteTypeHeader kind={kind}>
+        <h6>{KindToTitle.get(kind) ?? kind}</h6>
+        <div>
+          <TextInlineMedium bold>{isDefined(count) ? count : '-'}</TextInlineMedium>{' '}
+          <TextInlineMedium lighter>vote{plural(count)}</TextInlineMedium>
+        </div>
+        <DropDownButton isDropped={isOpen} onClick={toggle} />
+      </VoteTypeHeader>
+
+      <DropDownToggle isDropped={isOpen}>
+        {votes?.map(({ voter }, index) => (
+          <VoteListItem key={index}>
+            <MemberInfo key={index} member={voter} memberSize="s" />
+
+            <ButtonGhost size="small" onClick={() => undefined}>
```

Comment:
> @thesan point_up Am I right here, is there anything else we'd need to handle for the voting part of rationale?

The vote details including its rationale should be shown in a [modal](https://www.figma.com/file/pgNl83wgAub186krVkdiuo/Proposals?node-id=209%3A955), but I'm thinking about doing this in another PR